web-ui/index.html

```diff
@@ -12,7 +12,7 @@
   <title>Amazon IVS - Clip demo</title>
 
   <link rel="stylesheet" href="style.css">
-  <script src="https://player.live-video.net/1.8.0/amazon-ivs-player.min.js"></script>
+  <script src="https://player.live-video.net/1.23.0/amazon-ivs-player.min.js"></script>
```

Comment:
The latest version of the Player SDK is [1.24.0](https://docs.aws.amazon.com/ivs/latest/LowLatencyUserGuide/release-notes.html#jan18-24) - please upgrade to this version.

Reply:
updated Player SDK version to 1.23.0